cardstack/cardstack
# Show newly created cards

Closes #1002 

## Added:
- `cardLocalStorage` service to persist the ids of recent cards to local storage (get, add, remove, clear)
- unit tests for the service
- acceptance tests

I named the local storage service and methods in such a way that we can add more types of things to local storage in the future.

## Changed
- the model hook returns recent cards, and does not use `allCardsInStore` anymore
- in deployment, templates such as "event-card" will not show up in the "Recent Cards" list. This is a good thing, because we don't want our users to edit those directly, only adopt from them.

## Not included:
- I couldn't actually refresh the page in an acceptance test, so the service unit tests will have to do.
- there's no query param for clearing the local storage, since it's easier to do via command line. `    localStorage.removeItem('recentCardIds');`

## How to test manually

Create a card. Create another card. Go to the catalog page. Do you see the cards? Refresh. Are they still there? Cool!
